lib/tasks/test_track_rails_client_tasks.rake

```diff
@@ -10,6 +10,17 @@ namespace :test_track do
     end
   end
 
+  desc 'Generates build timestamp'
+  task generate_build_timestamp: :environment do
+    cli = TesttrackCli.instance
+    next if cli.skip_testtrack_cli?
+
+    if cli.project_initialized?
+      result = cli.call('generate_timestamp')
```

Comment:
Hmm, the naming of this command and the rake task is inconsistent. I like the more explicit name of the rake task, so probably should change the name of this command to `generate_build_timestamp`.

Reply:
so update the command in the cli?